front/components/home/ContentBlocks.tsx

```diff
@@ -142,7 +142,7 @@ export const HeaderContentBlock = ({
   <Grid>
     <div
       className={classNames(
-        "sm:pt-18 flex flex-col justify-end gap-12 pt-12 lg:pt-36",
+        "flex flex-col justify-end gap-12 pt-12 lg:pt-12",
```

Comment:
no need to duplicate the pt-12

```suggestion
        "flex flex-col justify-end gap-12 pt-12",
```